docs/time_series_causal_graph.md

```diff
@@ -0,0 +1,373 @@
+}
+
+# Simply via the adjacency matrices
+ts_cg = TimeSeriesCausalGraph.from_adjacency_matrices(adjacency_matrices=adjacency_matrices)
+
+# Also specifying the variable names
+ts_cg = TimeSeriesCausalGraph.from_adjacency_matrices(adjacency_matrices=adjacency_matrices, variable_names=['X', 'Y', 'Z'])
+```
+
+Let the nodes of this example be X,Y,Z. The corresponding edges are the following:
+edges = [
+...     (Z-2, Y),
+...     (X-1, X),
+...     (Y-1, X),
+...     (Y-1, Y),
+...     (Z-1, Z),
+...     (X, Y),
+...     (X, Z),
+...     (Y, Z),
+... ]
+
+## Minimal graph
+The minimal graph is the graph with the minimal number of edges that is equivalent to the original graph.
+In other words, it is a graph that has no edges whose destination is not time delta 0.
+
+
+```python
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+# Get the minimal graph
+minimal_graph = ts_cg.get_minimal_graph()
+
+# Check whether a graph is in its minimal form
+# Returns True if the graph is a minimal graph, False otherwise
+is_minimal = ts_cg.is_minimal_graph()
+```
+
+## Summary graph
+You can collapse the graph in time into a single node per variable (column name).
+This can become cyclic and bi-directed as X(t-1) -> Y and Y(t-1) -> X would become X <-> Y.
+Note that the summary graph is a `cai_causal_graph.causal_graph.CausalGraph` object.
+
+```python
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+
+summary_graph = ts_cg.get_summary_graph()
+```
+
+## Example
+Time series causal graph.
+X1(t-1) -> X1(t)
+X2(t-1) -> X2(t)
+X1(t-1) -> X2(t-1)
+X1(t) -> X2(t)
+X1(t) -> X1(t+1)
+X2(t) -> X2(t+1)
+X1(t+1) -> X2(t+1)
+![ts_cg](images/ts_cg.png)
+
+Minimal graph.
+X1(t-1) -> X1(t)
+X2(t-1) -> X2(t)
+X1(t-1) -> X2(t-1)
+X1(t) -> X2(t)
+![ts_minimal_graph](images/ts_minimal_graph.png)
+
+Summary graph.
+X1 -> X2
+![ts_summary_graph](images/ts_summary_graph.png)
+
+Extended graph with `backward_steps` = 2 and `forward_steps` = 0.
+X1(t-2) -> X1(t-1)
+X2(t-2) -> X2(t-1)
+X1(t-2) -> X2(t-2)
+X1(t-1) -> X1(t)
+X2(t-1) -> X2(t)
+X1(t-1) -> X2(t-1)
+X1(t) -> X2(t)
+![ts_cg](images/ts_extended_graph.png)
+
+
+## Extended graph
+You can extend the graph in time by adding nodes for each variable at each time step from `backward_steps` to
+`forward_steps`. If a backward step of n is specified, it means that the graph will be extended in order to
+include nodes back to time -n and all the nodes connected to them as specified by the minimal graph.
+If both `backward_steps` and `forward_steps` are None, the original graph is returned.
+
+```python
+from cai_causal_graph import TimeSeriesCausalGraph
+
+ts_cg: TimeSeriesCausalGraph
+backward_steps = 2
+forward_steps = 3
+
+extended_graph = ts_cg.extend_graph(backward_steps=backward_steps, forward_steps=forward_steps)
+```
+
+## CPDAGS
+`cai_causal_graph.causal_graph.TimeSeriesCausalGraph` also supports CPDAGs, as in the following example.
+
```

Comment:
@andrew-causalens What explanation should we give here?

Reply:
That they can encode Markov equivalence class of time series causal graphs. It is not just CPDAGs but MAGs and PAGs. Would rename the heading to Markov Equivalence Classes and then use a CPDAG as the example. But mention one could create a MAG or PAG too.